### DISCUSSION TITLE
Open up Transmog to white and grey items

### DISCUSSION THREAD

**Discussion by Vyperhand** [chromiecraft/chromiecraft#5661] [Suggestions]
Title - The module is already loading all grey and white items into the appearance collection. Would be very fun to have those available to use as xmog skins.  Some of the holiday items, and even some of the starter greys are very nice looking and would be very enjoyable to put on as appearances.

Also, there is Fun To Be Had putting on overalls and a shovel or pitchfork to go defend the farm in AB.  ;)

**Comment by 55Honey** [comment]
Hi Vyperhand,
thanks for your suggestion.

We keep looking into improvements for our existing systems and the implementation of new ideas all over. The transmog system is a regular part of this review as well.

Make sure you catch up with the announcements on our discord to be up to date about potential news.

Cheers!

**Reply by Vyperhand** [reply]
No worries Honey.  I have learned to share my ideas and bug reports Everywhere(TM).  ^_^   And thanks.

**Comment by Hogofogo1** [comment]
Absolutely agree with Vyper, 
availability of the gray and white items would not intrude any of the existing "same armor type" policy 
and would make it way easier to find matching parts.

**Comment by UnePomme** [comment]
Adding on to this: Event white items such as the pilgrim outfit or lunar dresses would be amazing to have as a tmog option. Some of the best looking tmogs are White event items.

**Reply by TotallyNotMyAccount** [reply]
I mean, do we really want orc warlocks running around in white wedding dresses? Or people using Last Year's Mutton as a weapon Tmog?
